qa/L0_input_validation/test.sh

```diff
@@ -64,14 +65,22 @@ set +e
 python3 -m pytest --junitxml="input_validation.report.xml" $TEST_PY >> $CLIENT_LOG 2>&1
 
 if [ $? -ne 0 ]; then
-    echo -e "\n***\n*** python_unittest.py FAILED. \n***"
+    echo -e "\n***\n*** input_validation_test.py FAILED. \n***"
     RET=1
 fi
 set -e
 
 kill $SERVER_PID
 wait $SERVER_PID
 
+pip install torch
```

Comment:
Why pytorch? Should be more flexible using Python backend

Reply:
This is taken from [this reproducer](https://github.com/triton-inference-server/server/issues/7171#issuecomment-2099531946)